src/types/composer.ts

```diff
@@ -513,8 +527,10 @@ export class TransactionComposer {
   /** The ATC used to compose the group */
   private atc = new algosdk.AtomicTransactionComposer()
 
-  /** Map of txid to ABI method */
-  private txnMethodMap: Map<string, algosdk.ABIMethod> = new Map()
+  /** Map of transaction index in the atc to a max logical fee.
+   * This is set using the value of either maxFee or staticFee.
+   */
+  private txnMaxFees: Map<number, AlgoAmount> = new Map()
```

Comment:
Checking: did you check all usages of the other similar `Map` to make sure you got all the places? There's quite a few code paths in this class and it's easy to miss stuff.

Reply:
I'll double check again, however I did check fairly extensively.
Worth noting that I haven't exposed the `coverAppCallInnerTransactionFees` option via the deprecated stateless functions to reduce the surface area.